### ISSUE TITLE
outook for ios app

### ISSUE THREAD

**Issue by Weychris** [MicrosoftDocs/OfficeDocs-Exchange#395]
please note that these group calendars not accessible through outlook app on ios

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cb586432-331c-fff4-953d-6e2b9d33f611
* Version Independent ID: 0e58d7c8-6d89-f854-cb8f-5a19c4169020
* Content: [Migrate your public folders to Office 365 Groups](https://docs.microsoft.com/en-us/exchange/collaboration/public-folders/migrate-to-office-365-groups?view=exchserver-2019)
* Content Source: [Exchange/ExchangeServer/collaboration/public-folders/migrate-to-office-365-groups.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeServer/collaboration/public-folders/migrate-to-office-365-groups.md)
* Product: **exchange-server-it-pro**
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by nenonix** [comment]
@Weychris  Thank you for the feedback. Do you think this can be addressed with a change to the content wording or by adding a note? If so I can work with you to reflect this in the documentation.

**Comment by Thomas-Titan** [comment]
Nicole,

In this article you are changing the Exchange default 14 day deletion to 30 days. I am running your commands (from this article) and they seem to be completing successfully except I get this warning message when I run this command “Get-Mailbox -ResultSize unlimited -Filter {(RecipientTypeDetails -eq 'UserMailbox')} | Set-Mailbox -RetainDeletedItemsFor 10”

WARNING: The command completed successfully but no settings of 'titan.local/MyBusiness/Users/SBSUsers/Tom Phillips'
have been modified.

I am trying to reduce the “day” to deletion to 10 days (or less if I can). We are planning to migrate our Exchange to a new server and I’m asking users to slim down there current Inbox so this migration time is minimized.

Why do I get the “Warning”?

Thanks,
Tom Phillips

**Comment by Weychris** [comment]
yes a note makes sense...maye simply a link to this page?
https://support.office.com/en-us/article/calendar-sharing-in-office-365-b576ecc3-0945-4d75-85f1-5efafb8a37b4?ui=en-US&rs=en-US&ad=US

**Comment by nenonix** [comment]
@Weychris  I agree that is a good solution. Would it work for you to add the link into the text under the Shared Calendars bullet point?
